src/KeyChain.m

```diff
@@ -19,8 +19,6 @@
 //
 
 #import "KeyChain.h"
-#import <Security/SecKeychain.h>
```

Comment:
Are you sure we dont need to `@import Security;` for `SecKeychainFindInternetPassword` etc?